src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+import Icon from '../Icon/Icon';
+/**
+ * The dropdown menu allow user to customize the button text, icon and action.
+ */
+const Dropdown = ({ className, items, topIcon, topText }) => {
+  const dropdownClass = classes('Dropdown', className);
+  return (
+    <div className={dropdownClass}>
+      <button type="button" className="Dropdown--button">
```

Comment:
for ♿️ add `tabIndex="0"` this will allow the button to be focusable with the keyboard 
https://bitsofco.de/how-and-when-to-use-the-tabindex-attribute/